apps/settings/js/modules/addon_manager.js

```diff
@@ -14,6 +14,7 @@ define(function(require) {
   'use strict';
 
   var AppsCache = require('modules/apps_cache');
+  var JSZip = require('modules/jszip');
```

Comment:
This will make JSZip get loaded by default but it is actually being used in the rename case. Let's load it dynamically in the rename function, please check my comments below.